docs/releases.asciidoc

```diff
@@ -6,6 +6,89 @@
 All notable changes for Debezium releases are documented in this file.
 Release numbers follow http://semver.org[Semantic Versioning].
 
+[[release-0-7-0]]
+== *Release 0.7.0* _(December 15th, 2017)_
+
+See the https://issues.jboss.org/secure/ReleaseNote.jspa?version=12335366&projectId=12317320[complete list of issues].
+
+=== Kafka compatibility
+
+This release has been built against Kafka Connect 1.0.0 and has been tested with version 1.0.0 of the Kafka brokers.
+See the https://kafka.apache.org/documentation/#upgrade[Kafka documentation] for compatibility with other versions of Kafka brokers.
+
+=== Upgrading
+
+Before upgrading the MySQL, MongoDB, or PostgreSQL connectors, be sure to check the backward-incompatible changes that have been made since the release you were using.
+
+When you decide to upgrade one of these connectors to 0.7.0 from any of the earlier 0.6.x, 0.5.x, 0.4.x, 0.3.x, 0.2.x, or 0.1.x versions,
+first check the upgrading notes for the version you're using.
+Gracefully stop the running connector, remove the old plugin files, install the 0.7.0 plugin files, and restart the connector using the same configuration.
+Upon restart, the 0.7.0 connectors will continue where the previous connector left off.
+As one might expect, all change events previously written to Kafka by the old connector will not be modified.
+
+=== Breaking changes
+
+This release includes the following changes that can affect existing installations:
+
+* Change default setting for BIGINT UNSIGNED handling https://issues.jboss.org/browse/DBZ-461[DBZ-461];
+`UNSIGNED BIGINT` is treated by default as `int64`, not as `Decimal` before. Set `bigint.unsigned.handling.mode` if you need to continue with the original behaviour.
+* Invalid value for HourOfDay ConnectException when the value of MySQL TIME filed is above 23:59:59 https://issues.jboss.org/browse/DBZ-342[DBZ-342]; The `TIME` datatype is mishandled by MySQL JDBC driver as it can contain not only time of day but arbitrary time length even greate then 24 hours. *...TBD...*
+* Postgres connectors stops to work after concurrent schema changes and updates https://issues.jboss.org/browse/DBZ-379[DBZ-379]; PostgreSQL connector was using JDBC metadata to get additional type information when it was processing logical events.
+This could lead to a race condition when database schema was updated and Debezium was still processing events with old schema structure.
+To mitigate the problem a new version of https://github.com/debezium/postgres-decoderbufs[Protocol Buffers decoder plugin] was introduced that passes additional type metadata with each event.
+The connector is backward compatible with the old decoder plugin (using the original approach) but we strongly recommend to upgrade it to the latest one as soon as possible.
+The race condition issue can still happen when primary key structure is changed for the table as this information is still obtained from JDBC metadata.
```

Comment:
We should describe how to avoid it (don't perform writes to the DB so the connector can consume all pending changes before doing such update). And the issue about changed column lengths etc. (unless you've described that somewhere else)?